src/tbp/monty/frameworks/environments/two_d_data.py

```diff
@@ -81,11 +81,11 @@ def __init__(self, patch_size=10, data_path=None):
         self.rotation = qt.from_rotation_vector([np.pi / 2, 0.0, 0.0])
         self.step_num = 0
         self.state = 0
-        self.data_path = data_path
-        if self.data_path is None:
-            self.data_path = os.path.join(os.environ["MONTY_DATA"], "omniglot/python/")
-        data_path = Path(self.data_path)
-        alphabet_path = data_path / "images_background"
+        if data_path is None:
+            self.data_path = Path(os.environ["MONTY_DATA"]) / "omniglot/python"
+        else:
+            self.data_path = Path(data_path)
```

Comment:
make utility function to replace these 4 lines

Reply:
It's a good idea, and I appreciate you deciding to separate it out into a separate change. 😀